packaging/tags.py

```diff
@@ -430,15 +435,42 @@ def mac_platforms(version=None, arch=None):
         arch = _mac_arch(cpu_arch)
     else:
         arch = arch
-    for minor_version in range(version[1], -1, -1):
-        compat_version = version[0], minor_version
-        binary_formats = _mac_binary_formats(compat_version, arch)
-        for binary_format in binary_formats:
-            yield "macosx_{major}_{minor}_{binary_format}".format(
-                major=compat_version[0],
-                minor=compat_version[1],
-                binary_format=binary_format,
-            )
+
+    if (10, 0) <= version and version < (11, 0):
+        # Prior to Mac OS 11, each yearly release of Mac OS bumped the
+        # "minor" version number.  The major version was always 10.
+        for minor_version in range(version[1], -1, -1):
+            compat_version = 10, minor_version
+            binary_formats = _mac_binary_formats(compat_version, arch)
+            for binary_format in binary_formats:
+                yield "macosx_{major}_{minor}_{binary_format}".format(
+                    major=10, minor=minor_version, binary_format=binary_format
+                )
+
+    if version >= (11, 0):
+        # Starting with Mac OS 11, each yearly release bumps the major version
+        # number.   The minor versions are now the midyear updates.
+        for major_version in range(version[0], 10, -1):
+            compat_version = major_version, 0
+            binary_formats = _mac_binary_formats(compat_version, arch)
+            for binary_format in binary_formats:
+                yield "macosx_{major}_{minor}_{binary_format}".format(
```

Comment:
"Mac OS X" was the name of the 10.x series. It was changed to "macOS" in 2016. Maybe it's time we do the switch for the tag too starting with 11.x? (could it somehow break backwards compatibility?)

Reply:
Changing the tag format would probably require a PEP.

Reply:
I'm against changing the tag. This is bound to break stuff and has limited advantages. 

In hindsight we should have set the tag to "macos" when we introduced this, but at that time "macosx" was the better choice (esp. because Mac OS 9 was still alive at that time).